src/components/Misc/RsvProgressBar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  current: string;
+};
+
+const states = {
+  idea: 'Idee',
+  agreement_process: 'Prüfung',
+  planning: 'Entwurf',
+  in_progress: 'Umsetzung',
+  done: 'Abgeschlossen',
+};
+
+const baseStyle =
+  'inline-flex items-center justify-center py-3 px-3.5 lg:px-2.5 text-xs lg:text-xs sm:text-base xl:py-2.5 xl:px-4 font-bold mr-1 xl:text-lg hover:cursor-default';
+const selectedStyle = 'bg-slate-900 text-white';
+const unselectedStyle = 'bg-gray-300 text-gray-600';
```

Comment:
Interessehalber: Warum hast du das out of scope und nicht in der function?

Ich würde die baseStyles direkt inline machen unten; und die beiden const dann direkt oberhalb von `const style = classNames`, damit man es auf einen Blick lesen kann.